flow/raster_cache.cc

```diff
@@ -84,19 +114,21 @@ std::unique_ptr<RasterCacheResult> RasterCache::Rasterize(
   }
 
   auto image = DlImage::Make(surface->makeImageSnapshot());
-  return std::make_unique<RasterCacheResult>(image, context.logical_rect,
-                                             context.flow_type);
+  return std::make_unique<RasterCacheResult>(
```

Comment:
I just noticed that the checkerboard will not contribute to the rtree. I'm not sure if we care, but if we do then we can just `rtree = nullptr` when there is a checkerboard.

Reply:
Perhaps having the checkerboard be sliced up by the rtree would provide an important debugging aid so developers can see exactly which parts of the overlay are being imposed upon? It might look odd, but it would provide information that isn't normally visible.

On the other hand, perhaps we might consider adding an overlay checkerboard feature specific to how the platform view slices are divided and recombined in the stack of platform views? Note that the rendering of the cached DL over the platform view might not represent the actual real estate consumed because the embedders often combine the rtree results into fewer larger rectangles, so slicing it up here doesn't give an accurate picture of how much real estate is being coopted...

Reply:
I'm leaning towards leaving this as it is. I have bad feeling about changing the behavior (disabling the R-Tree) upon a debug flag. It this turns out problematic we can reconsider.

Reply:
Note that we flush the cache when the flag changes, so this would only be modified while they have the mechanism enabled. But I agree that nothing needs to be done here and we can reevaluate our debugging tools later.